pages/loginpage.html

```diff
@@ -0,0 +1,115 @@
+            type="text"
+            placeholder="Search Products"
+          />
+        </div>
+        <li class="list-inline">
+          <a class="links link-active" href="./index.html">
+            <div class="badge-container">
+              <div class="badge red-icon">3</div>
+              <i class="fa-solid fa-heart cart-color"></i>
+            </div>
+          </a>
+        </li>
+        <li class="list-inline">
+          <a class="links link-active" href="./index.html">
+            <div class="badge-container">
+              <div class="badge red-icon">3</div>
+              <i class="fa-solid fa-cart-shopping cart-color"></i>
+            </div>
+          </a>
+        </li>
+        <li class="list-inline">
+          <a class="links link-active" href="./index.html">Products</a>
+        </li>
+        <li class="list-inline">
+          <a class="links" href="./Documentation/button/button.html">Login</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="flex-col login-container gap-sm">
+      <h2 class="login-header">Login</h2>
+      <label class="login-email" for="login-email">Email Address</label>
+      <input type="email" name="email" id="login-email" class="email-input" />
+      <label class="login-email" for="login-password">Password</label>
+      <input
+        type="password"
+        name="password"
+        id="login-email"
+        class="email-input"
+      />
+      <div class="show-password">
+        <input
+          type="checkbox"
+          name="checkbox"
+          class="show-pass-input"
+          id="showpassword"
+        />
+        <label class="show-pass-input" for="showpassword">Show password</label>
+        <a class="forgot-password" href="#">
+          <span>Forgot password?</span>
+        </a>
+      </div>
+      <button class="btn btn-primary-light">
+        <a class="login-label" href="#">Login</a>
+      </button>
+      <small class="create-acc-text">
+        Not a user yet?
+        <a class="create-acc-head" href="#">Create account</a>
+      </small>
+    </div>
+    <footer class="footer">
+        <div class="centered container footer-bottom">
+          <div class="footer-header">
+            <ul class="list-non-bullet">
+              <li class="list-inline"><a class="links" target="_blank" href="https://twitter.com/Priyanshicodes">Terms and conditions</a></li>
+              <li class="list-inline"><a class="links" target="_blank" href="https://twitter.com/Priyanshicodes">Shipping & Returns</a></li>
+              <li class="list-inline"><a class="links" target="_blank"  href="https://github.com/Priyanshi-Tiwari">Contact us</a></li>
+          </div>
+          <ul class="list-non-bullet">
+            <li class="list-inline">
+              <a
+                class="links"
+                target="_blank"
+                href="https://github.com/Priyanshi-Tiwari"
+                ><i class="fa-brands fa-github" style="font-size: 40px"></i
```

Comment:
Avoid applying inline styles you can use a class for it and use it here in the HTML